pkgs/dart_model/lib/src/json_buffer/typed_map.dart

```diff
@@ -187,6 +236,13 @@ extension TypedMaps on JsonBufferBuilder {
       void addBit(bool bit) {
         if (bit) byte += bitmask;
         bitmask <<= 1;
+
+        if (bitmask == 0x100) {
+          _setByte(valuePointer, byte);
```

Comment:
byte can be two bytes now right? does _setByte need to be renamed, does it actually work for 2 bytes?

Reply:
Added a comment clarifying

```
// On reaching bit 9 of `byte`, write the byte and reset `byte` and
// `bitmask`.
```

Reply:
Oh right, I was thinking `0x100` was 16 bytes. We really need binary integer literals lol.